packages/microsoft-reactnative-sampleapps/windows/SampleLibraryCPP/CircleViewManagerCPP.cpp

```diff
@@ -38,7 +38,7 @@ FrameworkElement CircleViewManagerCPP::CreateView() noexcept {
 
 // IViewManagerWithChildren
 
-void CircleViewManagerCPP::AddView(FrameworkElement const &parent, UIElement const &child, int64_t index) noexcept {
+void CircleViewManagerCPP::AddView(FrameworkElement const &parent, UIElement const &child, int64_t /*index*/) noexcept {
```

Comment:
Did you add this change as part of this PR? If so, could we separate the unrelated changes out? If not I think your branch might be in a weird state.

Reply:
I have eliminated a warning caused by the unused variable. My branch is in a good state.